data-source.ts

```diff
@@ -0,0 +1,15 @@
+import { DataSource } from 'typeorm';
+
+export const typeOrmConnectionDataSource = new DataSource({
```

Comment:
Do we know if these values are overridden when deployed to an environment?

Reply:
All of find seems to work with this connection config (on the main branch) - I'm not yet sure how this works but you can see the config used everywhere eg:

https://github.com/cabinetoffice/gap-find-backend/blob/main/ormconfig.json
https://github.com/cabinetoffice/gap-find-backend/blob/develop/.github/workflows/feature.yml
https://github.com/cabinetoffice/gap-find-web/blob/1457f47f3073a37aa708cd7d5437f8881afaa70a/.github/workflows/feature.yml#L19

Reply:
Nice one!

Reply:
I've set the datasource up so it gets values from `.env`. This file should now be populated with real values in all envs (setup through the find ECS task). The migrations work with this setup using a local .env.

I think we should check migrations still work in develop by accessing the infra directly first. We can attempt to add to each pipelines after we confirm that. Might be best to do the pipeline automation on another ticket - it seems like a nice to have as this ticket is blocking existing ci